cucumber-expressions/go/default.mk

```diff
@@ -88,7 +88,7 @@ dist_compressed/$(LIBNAME)-%: dist/$(LIBNAME)-%
 .tested: .deps $(GO_SOURCE_FILES) .go-tested
 
 .go-tested:
-	go test ./...
+	go test ./... -race
```

Comment:
All of the `**/go/default.mk` files in this monorepo are copies of `/.templates/go/default.mk`. This is the file you should modify. Then run `source scripts/functions.sh && rsync_files` from root to propagate the change to all the sub directories.